lib/idx/util.ts

```diff
@@ -193,6 +224,11 @@ export function getRemediator(
     // maybe return for next step
     remediatorCandidates.push(remediator);  
   }
+
+  // If no remedition is picked, use the first one with GenericRemeditor for default flow
+  if (!remediatorCandidates.length && !!idxRemediations.length && useGenericRemediator) {
+    return new GenericRemediator(authClient, idxRemediations[0], values, options);
```

Comment:
nit: call `remediator.canRemediate()` here before returning

Reply:
This logic happen in `getRemediator` to guarantee that there will always be an available remediator when `useGenericRemediator `. 

`canRemedidate` will be called afterward in `remediate` function.